can_spec_my18.yml

```diff
@@ -531,22 +555,31 @@ buses:
             c_type: int32_t
             signed: false
             unit: Wh
-      DashRequest:
+      DashControls:
         can_id: 0x0D2
         is_big_endian: true
         segments:
-          type:
+          using_regen:
             position: 0
-            length: 3
-            c_type: enum
-            enum:
-              NO_REQUEST: 0x0
-              TRACTION_CONTROL_ENABLE: 0x5
-              TRACTION_CONTROL_DISABLE: 0x6
-              ACTIVE_AERO_ENABLE: 0x7
-              ACTIVE_AERO_DISABLE: 0x8
-              FAN_ENABLE: 0x9
-              FAN_DISABLE: 0xA
+            length: 1
+            c_type: bool
+          regen_bias: # Ranges from 0 to 100
+            position: 1
+            length: 16
```

Comment:
Too big?

Reply:
Yeah, I was on the fence about this...but I was thinking about our conversation the other day about how it benefit vs. cost of using a larger integer size

Reply:
Given your other comment I suppose I can squish them

Reply:
Don't squish unless you plan on reflashing dash. Maybe squish if and when we need it.